stable/jenkins/templates/config.yaml

```diff
@@ -116,6 +116,10 @@ data:
               <imagePullSecrets/>
 {{- end }}
               <nodeProperties/>
+{{- if .Values.agent.yamlTemplate }}
+              <yaml>
+{{ .Values.agent.yamlTemplate | replace "\"" "&quot;" | indent 4 }}</yaml>
```

Comment:
- Did you think about other characters which need to be escaped?
- Is there a reason why you have choosen an indent of 4? Does it look better in the Jenkins UI?
- I would move the </yaml> to the next line.
- You could also run the yamlTemplate through the [tpl](https://github.com/helm/helm/blob/master/docs/charts_tips_and_tricks.md#using-the-tpl-function) that woul allow users to reference other values.

Reply:
I looked for escaping functions but didn't find anything useful but I'm no expert. The one replacement there is copied from the previous PR.

The 4 char indent is confusing because it is only there to preserve the outer yaml indent so the resulting xml file has no indent

I experimented some with using pure yaml but was unable to get it working but I wasn't using tpl so I can try that

Reply:
Ah ok it looks like `html` was what I needed.